index.html

```diff
@@ -5,12 +5,45 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="style.css">
     <title>API Practice</title>
 </head>
 
 <body>
-    <div class="container"></div>
-    <button id="random">Get Info</button>
+
+    <button id="random" onclick="getCharacters()">Get Info</button>
+    <div class="container">
+
+       
+        <div class='item'> 
+            <p id='name'>NAME</p>
```

Comment:
you can make it so each item is automatically made in your JS file instead of manually adding each item one at a time.